src/voice_registration.js

```diff
@@ -34,191 +35,615 @@ go.app = function() {
             var speech_option = '1';
             return new ChoiceState(name, {
-                question: $('Pregnant or baby'),
+                question: $('Last period?'),
                 helper_metadata: go.utils.make_voice_helper_data(
                     self.im, name, lang, speech_option),
                 choices: [
-                    new Choice('pregnant', $('Pregnant')),
-                    new Choice('baby', $('Baby'))
+                    new Choice('state_5A_period_month', $('This year')),
+                    new Choice('state_5B_period_month', $('Last year'))
                 ],
-                next: 'state_r05_birth_year'
+                next: function(choice) {
+                    return choice.value;
+                }
             });
         });
 
-        self.add('state_r05_birth_year', function(name) {
-            // TODO #6 Don't show next year for pregnancy in Jan / Feb
-            var speech_option;
-            var year_choices = [
-                new Choice('last_year', $('last_year')),
-                new Choice('this_year', $('this_year')),
-                new Choice('next_year', $('next_year'))
-            ];
-            if (self.im.user.answers.state_r04_mom_state === 'pregnant') {
-                choices = year_choices.slice(1,3);
-                speech_option = '1';
-            } else {
-                choices = year_choices.slice(0,2);
-                speech_option = '2';
-            }
+        // ChoiceState st-5A
+        self.add('state_5A_period_month', function(name) {
+            var speech_option = 1;
+
             return new ChoiceState(name, {
-                question: $('Birth year?'),
+                question: $('Period month?'),
                 helper_metadata: go.utils.make_voice_helper_data(
                     self.im, name, lang, speech_option),
-                choices: choices,
+                choices: [
+                    new Choice('jan', $('January')),
+                    new Choice('feb', $('February')),
+                    new Choice('mar', $('March')),
+                    new Choice('apr', $('April')),
+                    new Choice('may', $('May')),
+                    new Choice('jun', $('June')),
+                    new Choice('jul', $('July')),
+                    new Choice('aug', $('August')),
+                    new Choice('sep', $('September')),
+                    new Choice('oct', $('October')),
+                    new Choice('nov', $('November')),
+                    new Choice('dec', $('December'))
+                ],
                 next: function(choice) {
-                    return 'state_r06_birth_month';
+                    var today = go.utils.get_today(self.im.config);
+                    var currentMonth = parseInt(today.format("MM"));
+                    var validStartMonth = currentMonth <= 10 ? 0 : currentMonth-10;
+                    var choiceMonth = parseInt(today.month(choice.value).format("MM"));
```

Comment:
This needs to move to a function and needs comments, but leave it for now if it works - open another issue to improve it.